src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * Deals with making sense of the user command.
+ */
+public class Parser {
+    private static String[] keywords = {"bye", "list", "mark", "unmark", "todo", "deadline", "event", "delete", "find"};
+
+    public Parser() {};
```

Comment:
The coding standard doesn't say about empty constructors so I'm not sure but I think it's 
```suggestion
    public Parser() {
    }
```
I think the semicolon is not needed as well.